.github/workflows/test.yml

```
@@ -0,0 +1,66 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        os: [
+          "ubuntu-18.04",
+          "macOS-10.14",
+        ]
+        python-version: [
+          "pypy3",
+          "3.7",
+          "3.5",
+          "3.6",
+        ]
+        include:
+        - python-version: "3.5"
+          env: PYTHONOPTIMIZE=2
+        - python-version: "3.6"
+          env: PYTHONOPTIMIZE=1
+    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.os }} Python ${{ matrix.python-version }}
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Linux dependencies
+      if: matrix.os == 'ubuntu-18.04'
+      run: |
+        .travis/install.sh
+
+    - name: Install macOS dependencies
+      if: matrix.os == 'macOS-10.14'
```

Untested, but what about this?
```suggestion
      if: matrix.os =~ ^macOS
```

---

Unexpected symbol: '=~':

`- Your workflow file was invalid: The pipeline is not valid. .github/workflows/test.yml (Line: 37, Col: 11): Unexpected symbol: '=~'. Located at position 11 within expression: matrix.os =~ ^ubuntu,.github/workflows/test.yml (Line: 42, Col: 11): Unexpected symbol: '=~'. Located at position 11 within expression: matrix.os =~ ^macOS`

https://github.com/hugovk/pillow/commit/6e73b1cc50ec987e67392c6514829b2ab6e6d97b/checks?check_suite_id=257734058

---

You can use the `contains` function: https://github.com/python-pillow/Pillow/blob/a0a560168908d2aed79439cd1ed7811ece3381bb/.github/workflows/test-windows.yml#L36
There is also a `startsWith` function. See the docs: https://help.github.com/en/articles/contexts-and-expression-syntax-for-github-actions#functions

---

Thanks, updated with `startsWith`.